baselines/fedstar/pyproject.toml

```diff
@@ -0,0 +1,138 @@
+[build-system]
+requires = ["poetry-core>=1.4.0"]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "fedstar" # <----- Ensure it matches the name of your baseline directory containing all the source code
+version = "1.0.0"
+description = "Flower Baselines"
+license = "Apache-2.0"
+authors = ["The Flower Authors <hello@flower.dev>"]
```

Comment:
you might want to add your contact details here. and update the `description` field above with the title of your baseline/paper

Reply:
Thanks for the suggestion I have updated the files accordingly.